core/src/test/java/org/frankframework/jdbc/dbms/DbmsSupportTest.java

```diff
@@ -687,38 +687,6 @@ public void testIsClobType() throws SQLException {
 		}
 	}
 
-	@DatabaseTest
```

Comment:
Just for my information, why can this be removed?

Reply:
Net uitgelegd, maar ik was nog niet klaar met typen van deze uitleg toen jij de vraag al stelde 😬

Reply:
Ja, je bood hem al ter review aan 😉

Reply:
Ja voordat ik een comment op een PR kan doen, moet ik eerst het PR maken. En ik had gelijk iedereen toegevoegd, ik dacht, zo snel kijkt toch niemand. 😂